ci-operator/step-registry/jboss-web-server/lp-interop-tests/jboss-web-server-lp-interop-tests-commands.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "Run JWS Interop testing..."
+SECRETS_DIR="/tmp/secrets/tests"
+mkdir -p  ${ARTIFACT_DIR}/jws_artifacts/jws-5
+export KUBECONFIG=$SHARED_DIR/kubeconfig
+
+export PATH=$PATH:/tmp/bin
+mkdir -p /tmp/bin
+curl -sL https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 > /tmp/bin/jq
```

Comment:
You could do this with inputs into the image. It would probably look something like[ this](https://github.com/openshift/release/blob/aa4d6f2feb51a04fe083844f64e5d4bbc93b02fb/ci-operator/config/mtc-qe/mtc-e2e-qev2/mtc-qe-mtc-e2e-qev2-master__mtc1.8-ocp4.15-lp-interop.yaml#L24-L30) (see that the `cli` image is being imported in `base_images` then the `oc` binary is being copied from `cli` to the new image). I think the image you would want is `cli-jq` the import for that would be something like:

```
base_images:
    cli-jq:
        namespace: ocp
        name: cli-jq
        tag: latest
```

All that being said, this also works. We will just always be using jq 1.6 where the process above would keep it up to date with the latest image.

Reply:
done